Assets/Scripts/GameData/IDescriptable.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SwordAndBored.GameData
+{
+    /// <summary>
+    /// An object that has a name and a description
+    /// </summary>
+    interface IDescriptable
+    {
+        string Name { get; set; }
+        string Description { get; set; }
```

Comment:
Will this name field be used to set a soldier's name, like Grand Mage Crawfis?

Reply:
Yes